src/app/shared/angular2-highlight-js/readme.md

```diff
@@ -0,0 +1,157 @@
+### angular2-highlight-js

```

Comment:
This looks like highlight js's readme. 

Please add a readme describing why we needed to embed highlightjs in our repo, which version you chose, and how it is modified from the normal version so that future developers can maintain or hopefully replace this embedded solution. 
Thanks!